scripts/tests/unit.sh

```diff
@@ -17,4 +17,5 @@ cd "$(git rev-parse --show-toplevel)"
 # tests from here. If you want to run other test suites, see the predefined
 # tasks in scripts/test.mk.
 
-echo "Unit tests are not yet implemented. See scripts/tests/unit.sh for more."
+make run-unit-tests
+echo "Unit tests are now implemented. See scripts/tests/unit.sh for more."
```

Comment:
This looks odd. I think it means `make test-unit` would end up running `make run-unit-tests`. Seems unnecessarily complicated.

Also the `echo` isn't helpful. We should show, not tell.

How about something like:

```suggestion
echo "Running unit tests"
pytest
```